src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -427,141 +435,104 @@ public void It_fails_for_unsupported_rid()
+            new DotnetCommand(Log)
                 .WithWorkingDirectory(slnDir)
-                .Execute(@"dotnet", "publish")
+                .Execute("dotnet", "publish")
                 .Should()
                 .Pass();
 
-            Environment.SetEnvironmentVariable(EnvironmentVariableNames.ENABLE_PUBLISH_RELEASE_FOR_SOLUTIONS, null);
-
-            var expectedAssetPath = System.IO.Path.Combine(slnDir, "App", "bin", "Release", ToolsetInfo.CurrentTargetFramework, "publish", "App.dll");
+            var expectedAssetPath = Path.Combine(slnDir, "App", "bin", "Release", ToolsetInfo.CurrentTargetFramework, "publish", "App.dll");
             Assert.True(File.Exists(expectedAssetPath));
 
         }
 
         [Fact]
         public void It_passes_using_PublishRelease_with_conflicting_capitalization_but_same_values_across_solution_projects()
         {
-            Environment.SetEnvironmentVariable(EnvironmentVariableNames.ENABLE_PUBLISH_RELEASE_FOR_SOLUTIONS, "true");
-
             var slnDir = _testAssetsManager
-               .CopyTestAsset("TestAppWithSlnUsingPublishReleaseConflictingCasing", "PublishReleaseConflictSln")
+               .CopyTestAsset("TestAppWithSlnUsingPublishReleaseConflictingCasing")
                .WithSource()
                .Path;
 
-            new BuildCommand(Log, slnDir, "App.sln")
-               .Execute()
-               .Should()
-               .Pass();
-
-            var publishCommand = new DotnetCommand(Log)
+            new DotnetCommand(Log)
                 .WithWorkingDirectory(slnDir)
-                .Execute(@"dotnet", "publish")
+                .Execute("dotnet", "publish")
                 .Should()
                 .Pass();
 
-            Environment.SetEnvironmentVariable(EnvironmentVariableNames.ENABLE_PUBLISH_RELEASE_FOR_SOLUTIONS, null);
-
-            var expectedAssetPath = System.IO.Path.Combine(slnDir, "App", "bin", "Release", ToolsetInfo.CurrentTargetFramework, "publish", "App.dll");
+            var expectedAssetPath = Path.Combine(slnDir, "App", "bin", "Release", ToolsetInfo.CurrentTargetFramework, "publish", "App.dll");
             Assert.True(File.Exists(expectedAssetPath));
-
         }
 
-
         [Fact]
-        public void It_warns_if_PublishRelease_set_on_sln_but_env_var_not_used()
+        public void It_no_longer_warns_if_PublishRelease_set_on_sln_but_env_var_not_used()
         {
             var slnDir = _testAssetsManager
-               .CopyTestAsset("TestAppWithSlnUsingPublishRelease", "PublishReleaseSlnNoEnvVar")
+               .CopyTestAsset("TestAppWithSlnUsingPublishRelease")
                .WithSource()
                .Path;
 
-            new BuildCommand(Log, slnDir, "App.sln")
-               .Execute()
-               .Should()
-               .Pass();
-
-            var publishCommand = new DotnetCommand(Log)
+            new DotnetCommand(Log)
                 .WithWorkingDirectory(slnDir)
                 .Execute(@"dotnet", "publish")
```

Comment:
Won't this essentially run `dotnet dotnet publish`?  It seems like we wouldn't want to pass "dotnet" as the first argument to `Execute`